src/agent/index.js

```diff
@@ -4079,7 +4079,7 @@ function fsList (args) {
 }
 
 function fsGet (args) {
-  return fs.cat(args[0] || '', '*');
+  return fs.cat(args[0] || '', '*', args[1] || 0, args[2] || null);
```

Comment:
```suggestion
  return fs.cat(args[0] ?? '', '*', args[1] ?? 0, args[2] ?? null);
```

(Moving to newer syntax while at it.)